src/chrome/content/rules/M-net.de.xml

```diff
@@ -1,9 +1,82 @@
 <ruleset name="M-net Telekommunikations GmbH" default_off="broken">
```

Comment:
So re-enabled.
I only could find a small mixed-content issue with a tracking element on the board. Shouldn't be an issue.